docs/docs/v10-upgrade-guidance.mdx

```diff
@@ -1,263 +1,145 @@
 
 ## CLI
 
 ### Removed command `cli reconsent`
 
-The `cli reconsent` command was removed. This command was removed due to the removal of the PnP Management Shell service application.
+Due to the removal of the PnP Management Shell app registration, the cli reconsent command had no use anymore. Therefore, we have removed this command from the CLI.
 
-## CLI configuration
+### Removed `showSpinner` setting
 
-### Removed setting `showSpinner`
-
-We removed the `showSpinner` config variable from the CLI configuration. Due to several issues, the spinner has been permanently removed.
+Persistent issues with the spinner led to its removal from the CLI a few months ago. However, the `showSpinner` setting remained in the CLI configuration. In this major release, we have removed the `showSpinner` setting entirely.
 
 #### What action do I need to take?
 
-Remove all entries in your script where you use the `showSpinner` setting.
-
-## CLI Doctor
+Remove all instances in your scripts where you reference the `showSpinner` setting.
 
 ### Updated output of `cli doctor` command
 
-For this new major version, we've adjusted the output of the `cli doctor` command to align the renaming of Azure AD to Microsoft Entra ID.
+In this major release, we have updated the [cli doctor](./cmd/cli/cli-doctor.mdx) command output to reflect the renaming of Azure AD to Microsoft Entra ID.
 
-The output is updated with `cliAadAppId` as `cliEntraAppId` and `cliAadAppTenant` as `cliEntraAppTenant`.
+The following properties have been renamed in the result:
 
-#### What action do I need to take?
+- `cliAadAppId` to `cliEntraAppId`
+- `cliAadAppTenant` to `cliEntraAppTenant`
 
-Please, check the documentation of the [cli doctor](./cmd/cli/cli-doctor.mdx) command to see the updated output and adjust your scripts accordingly.
+#### What action do I need to take?
 
-## CLI environment variables
+Review the documentation of the `cli doctor` command to see the new output structure and adjust your scripts accordingly.
 
-### Removed deprecated CLI environment variables
+### Removed deprecated environment variables
 
-For this new major version, we've removed the deprecated `AAD` CLI environment variables. We have renamed `CLIMICROSOFT365_AADAPPID` variable to `CLIMICROSOFT365_ENTRAAPPID`.
+In the aftermath of renaming all Azure AD-related commands and options to Microsoft Entra ID, we have removed the deprecated environment variable `CLIMICROSOFT365_AADAPPID` that was used to store the Microsoft Entra app ID.
 
 #### What action do I need to take?
 
-Please, check the user guidance [Log in to Microsoft 365](./user-guide/connecting-microsoft-365.mdx) command to see the updated environment variables to use.
-
-## CLI Status
+Instead of using `CLIMICROSOFT365_AADAPPID`, use `CLIMICROSOFT365_ENTRAAPPID` to store the Microsoft Entra app ID.
 
 ### Removed deprecated CLI environment variables
```

Comment:
I only had a quick glance but I think this title does not refer to status update. I think it is a copy paste mistake from the update done above